shakirandagire/RecipeSearch
# Feature #61743860(Sub-Task) Restructuring the application

What does this PR do?
Restructuring of the application.
*Description of Task to be completed?
Restructuring of the application to meet standards.
How to test the application
Clone and set up the project
Use $ git checkout ft-create-search-bar to check out the branch containing the changes of this pull request
Use $ npm start to start the application
Visit http://localhost:3000/.
Pivotal Tracker Story
[#161743860](https://www.pivotaltracker.com/story/show/161743860)
